### ISSUE TITLE
Decreased Mid-Arm Muscle Area

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5898]
**Preferred term label:** Decreased Mid-Arm Muscle Area

**Synonyms** Decreased MAMA

**Definition (free text, please give PubMed ID)** Estimation of the area of the mid-upper arm that is composed of muscle is decreased., PMID: 30558577

**Parent term (use hpo.jax.org/app)** Abnormality of the upper arm HP:0001454

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think it is sufficient to provide a term for the "circumference" since the area is a simple derived measure.
